Resources/public/js/models/ez-contentmodel.js

```diff
@@ -166,6 +166,8 @@ YUI.add('ez-contentmodel', function (Y) {
          * Filters the relations on this content by type or optionally by field
          * definition identifier.
          *
+         * Not specifying `type` and `fieldDefinitionIdentifier` will return all relations
```

Comment:
a better way to document that is to document the `type` parameter as optional and add such details in the doc of the parameter:

```js
* {String} [type] type of relation to filter on ('ATTRIBUTE', 'COMMON', 'EMBED', 'LINK'), if omitted all relations are returned
```